internal/client/state.go

```diff
@@ -32,12 +33,13 @@ type RawConfig struct {
 	RemotePort       string   // jsonOptional
 	AlternativeNames []string // jsonOptional
 	// defaults set in ProcessRawConfig
-	UDP           bool   // nullable
-	BrowserSig    string // nullable
-	Transport     string // nullable
-	CDNOriginHost string // nullable
-	StreamTimeout int    // nullable
-	KeepAlive     int    // nullable
+	UDP               bool   // nullable
+	BrowserSig        string // nullable
+	Transport         string // nullable
+	CDNOriginHost     string // nullable
+	CDNRequestUrlPath string // nullable
```

Comment:
Not sure if "Request" is needed in the name. Maybe `CDNWsUrlPath`?

Reply:
I'm totally no idea on naming variables and you decide it. I'll commit later, after all changes of `README.md` settles.